### ISSUE TITLE
ginina install error 

### ISSUE THREAD

**Issue by croshong** [gnina/gnina#250]
Hi 

I tried to install gnina from source

after cmake is done

I got error like below
[ 87%] Building CXX object gninasrc/CMakeFiles/gnina.dir/main/main.cpp.o
[ 88%] Linking CUDA device code CMakeFiles/gnina.dir/cmake_device_link.o
[ 88%] Linking CXX executable ../bin/gnina
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `fcntl64@GLIBC_2.28'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `powf@GLIBC_2.27'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `exp@GLIBC_2.29'
../_deps/libtorch-src/lib/libc10.so: undefined reference to `log@GLIBC_2.29'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `expf@GLIBC_2.27'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `logf@GLIBC_2.27'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `log2@GLIBC_2.29'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `pow@GLIBC_2.29'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `log2f@GLIBC_2.27'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `exp2@GLIBC_2.29'
/lwork01/gnina/build/_deps/libtorch-src/lib/libtorch_cpu.so: undefined reference to `exp2f@GLIBC_2.27'

I heard that gnina use caffe for its deep learning framework,

Is it updated to include pytorch model?

I have install pytorch but same error occur

how can I solve this problem?

Thanks

**Comment by dkoes** [comment]
The code in master currently uses pytorch and caffe with the goal of completely removing caffe in the next release.  If you just want to get a working version of gnina, checkout v1.1.

**Comment by croshong** [comment]
Yes but I want to try to install gnina from source to be optimized for my server . to install from source how can I solve the above error?

**Comment by dkoes** [comment]
`git checkout v1.1`
